setup.py

```diff
@@ -16,7 +16,7 @@
 
 setup(
     name="data_gateway",
-    version="0.8.0",
+    version="0.8.1",
```

Comment:
Multi threading feels like a new feature 😂

Reply:
Yeah I guess, I'm doing it to fix a bug though :P I think I'll probably change it to a minor version increase.